src/transport/RendezvousSession.cpp

```diff
@@ -29,10 +29,11 @@ static const char * kSpake2pKeyExchangeSalt        = "SPAKE2P Key Exchange Salt"
 
 namespace chip {
 
-CHIP_ERROR RendezvousSession::Init()
+CHIP_ERROR RendezvousSession::Init(const RendezvousParameters & params)
 {
     CHIP_ERROR err = CHIP_NO_ERROR;
 
+    mParams = params;
```

Comment:
Before the SetupPINCode was enforced by making sure the constructor receive something. Can we add a VerifyOrExit check with a HasSetupPINCode method in the RendezvousParameters ?
Also we can probably check that the value is not 0 since this one means the accessory is already paired.

Reply:
Oh I don't know pincode 0 means already paired, thanks for pointing out.